x-pack/plugins/aiops/public/components/spike_analysis_table/spike_analysis_table.tsx

```diff
@@ -78,56 +98,87 @@ export const SpikeAnalysisTable: FC<SpikeAnalysisTableProps> = ({
     },
     {
       field: 'pValue',
-      name: 'p-value',
+      name: (
+        <EuiToolTip
+          position="top"
+          content={i18n.translate(
+            'xpack.aiops.correlations.failedTransactions.correlationsTable.pValueColumnTooltip',
+            {
+              defaultMessage:
+                'For statistically significant changes in the frequency of values, indicates how extreme the change is; lower values indicate greater change.',
+            }
+          )}
+        >
+          <FormattedMessage
+            id="xpack.aiops.correlations.failedTransactions.correlationsTable.pValueLabel"
+            defaultMessage="p-value"
+          />
+        </EuiToolTip>
+      ),
       render: (pValue: number) => pValue.toPrecision(3),
       sortable: true,
     },
     {
       field: 'pValue',
       name: (
-        <>
-          {i18n.translate(
-            'xpack.aiops.correlations.failedTransactions.correlationsTable.impactLabel',
+        <EuiToolTip
+          position="top"
+          content={i18n.translate(
+            'xpack.aiops.correlations.failedTransactions.correlationsTable.impactLabelColumnTooltip',
             {
-              defaultMessage: 'Impact',
+              defaultMessage:
+                'Indicates the level of impact of the given field name and value on the message rate difference.',
             }
           )}
-        </>
+        >
+          <FormattedMessage
+            id="xpack.aiops.correlations.failedTransactions.correlationsTable.impactLabel"
+            defaultMessage="Impact"
+          />
+        </EuiToolTip>
       ),
       render: (_, { pValue }) => {
         const label = getFailedTransactionsCorrelationImpactLabel(pValue);
         return label ? <EuiBadge color={label.color}>{label.impact}</EuiBadge> : null;
       },
-      sortable: true,
+      sortable: false,
```

Comment:
Can sorting be enabled on the `Impact` field - it uses `pValue` under the hood, and is sortable in APM correlations, for example https://github.com/elastic/kibana/blob/main/x-pack/plugins/apm/public/components/app/correlations/failed_transactions_correlations.tsx

Reply:
Added in 2ce5f5ae8a7b23d8775d31d96797d497179af975